api/src/referenceApp.Common/Models/Entity/IdentityProviderEntityModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+namespace referenceApp.Common.Models.Entity
```

Comment:
What are the models in the "Common" project used for?  API models should go in the API project.  Persistence models in the Persistence project.